tools/docker/core/Dockerfile

```diff
@@ -42,5 +42,5 @@ RUN apk add --no-cache --virtual .build-deps \
         musl-dev \
         libffi-dev \
     && python tools/pipstrap.py \
-    && python tools/pip_install_editable.py src/acme src/certbot \
+    && PIP_NO_BUILD_CACHE=no python tools/pip_install_editable.py src/acme src/certbot \
```

Comment:
Is this environment variable right here and in the plugin Dockerfile? I was surprised that it doesn't match the name `--no-cache-dir` and there are no results for `PIP_NO_BUILD_CACHE` on Google.

If the variable name is wrong, we could fix that or we could just use `tools/pip_install.py` here and add `-e` and `--no-cache-dir` ourselves. I personally kind of prefer this since I think `pip` is silently ignoring this environment variable and shouldn't silently ignore flags it doesn't understand.

Reply:
The variable name is wrong. I was sure to take it right...

Reply:
Indeed, I forgot that `pip_install.py` is passing args as they are to pip.